test/tool/gen-slice.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+/* eslint guard-for-in: 0 */
+
+import path from 'path';
+import {slice} from '../../src/slice.js';
+import {Tensor} from '../../src/lib/tensor.js';
+import {utils} from './utils.js';
+
+(() => {
+  function sliceCompute(inputShape, inputValue, starts, sizes, options = {}) {
+    const inputTensor = new Tensor(inputShape, inputValue);
+    const outputTensor = slice(inputTensor, starts, sizes, options);
+    return outputTensor.data;
+  }
+
+  const min = -100.0;
+  const max = 100.0;
+  const savedDataFile = path.join(path.dirname(process.argv[1]), 'test-data', 'slice-data.json');
```

Comment:
Would `path.dirname(process.argv[1]), 'test-data'` be better as a parameter, rather than repeating it in each generator? It would be annoying to update a dozen different places if you decided to change the test folder path later. Or define a helper function that takes a data file name? e.g.

`const savedDataFile = getSavedDataFilePath('slice-data');`